cmd/fossa/cmd/report/dependencies.go

```diff
@@ -0,0 +1,41 @@
+package report
+
+import (
+	"text/template"
+
+	"github.com/fossas/fossa-cli/cmd/fossa/flags"
+	"github.com/fossas/fossa-cli/log"
+	"github.com/fossas/fossa-cli/pkg"
+	"github.com/urfave/cli"
+)
+
+var dependenciesCmd = cli.Command{
+	Name:  "dependencies",
+	Usage: "Generate dependencies report",
+	Flags: flags.WithGlobalFlags(flags.WithAPIFlags(flags.WithModulesFlags([]cli.Flag{
+		cli.StringFlag{Name: flags.Short(Output), Destination: &outputFlag, Value: "-", Usage: "Output file for report"},
```

Comment:
It's helpful to add flags used in multiple places to `cmd/fossa/flags`. It conveys the intent that these flags are identical, and prevents their usage strings from drifting out of sync.